modules/ROOT/pages/use-api-catalog-cli.adoc

```diff
@@ -0,0 +1,50 @@
+= Using API Catalog CLI
+
+After you have installed the API Catalog CLI and configured permissions, you can use the API Catalog CLI commands to autocatalog your APIs.
+
+//Command summary start
+
+[[api-catalog-commands]]
+== API Catalog Commands
+
+API Catalog CLI includes the following commands:
+
+include::partial$api-catalog.adoc[tag=summary]
+
+//Command summary end
+
+== View Command Help
+
+Use the help option to list the commands or command options and their descriptions. The following examples show how the help option is used:
```

Comment:
how the help option is used: > how to use the help option:
(suggestion to avoid passive voice)